src/calculateRentalCost.js

```diff
@@ -28,7 +28,18 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const dayRent = 40;
+  const bigDiscount = 50;
+  const smallDiscount = 20;
+  let total = dayRent * days;
+
+  if (days >= 7) {
+    total -= bigDiscount;
+  } else if (days >= 3) {
+    total -= smallDiscount;
+  }
```

Comment:
You should return value right away after you've found the right condition. It makes the code simpler to read and helps to avoid unnecessary `else if`

```suggestion
  if (days >= 7) {
    return total - bigDiscount;
  } 
  
  if (days >= 3) {
    return total - smallDiscount;
  }
```